packages/webdriver/src/native-fetch.ts

```diff
@@ -0,0 +1,6 @@
+export { RequestInfo, RequestInit, Response } from 'node-fetch'
+export { AbortController } from 'abort-controller';
+
+import fetch from 'node-fetch';
+
+export { fetch };
```

Comment:
Why is this in a separate file?

Reply:
I was trying to make the `fetch` code itself be agnostic about where the native fetch function was coming from so that we could use `@effection/fetch` in both the browser and node since in the browser, `fetch` is a global variable. Not sure the best way to do this in a multi-platform way, but figured that isolating the symbol definitions was a start.